aQute.libg/src/aQute/libg/command/Command.java

```diff
@@ -169,6 +169,8 @@ public int execute(final InputStream in, Appendable stdout, Appendable stderr) t
 				} else {
 					try {
 						IO.copy(in, stdin);
+					} catch (Exception e) {
+						// Who cares?
```

Comment:
Eating exceptions is generally not a good idea. At the least the exception should be logged or something.

Reply:
I've added a bit more logging when exceptions occur when copying stdin.